docs/openstack/edpm_adoption.md

```diff
@@ -310,3 +315,124 @@ EOF
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+## Nova compute services fast-forward upgrade from Wallaby to Antelope
+
+Nova services rolling upgrade cannot be done during adoption,
+there is in a lock-step with Nova control plane services, because those
+are managed independently by EDPM ansible, and Kubernetes operators.
+Nova service operator and OpenStack Dataplane operator ensure upgrading
+is done independently of each other, by configuring
+`[upgrade_levels]compute=auto` for Nova services. Nova control plane
+services apply the change right after CR is patched. Nova compute EDPM
+services will catch up the same config change with ansible deployment
+later on.
+
+> **NOTE**: Additional orchestration happening around the FFU workarounds
+> configuration for Nova compute EDPM service is a subject of future changes.
+
+* Wait for cell1 Nova compute EDPM services version updated (it may take some time):
+
+    ```bash
+    oc exec -it mariadb-openstack-cell1 -- mysql --user=root --password=${PODIFIED_DB_ROOT_PASSWORD} \
+      -e "select a.version from nova_cell1.services a join nova_cell1.services b where a.version!=b.version and a.binary='nova-compute';"
+    ```
+  The above query should return an empty result as a completion criterion.
+
+* Remove pre-FFU workarounds for Nova control plane services:
+
+    ```yaml
+    oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+    spec:
+      nova:
+        template:
+          cellTemplates:
+            cell0:
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=false
```

Comment:
here and below: you don't need an explicit `disable_compute_service_check_for_ffu=false` as false is the default. I suggest to just drop the content of `customServiceConfig` field.

Reply:
The edpm-nova config cleanup bug mentioned below does not effect the k8s control plane.

Reply:
how am I supposed to drop this section? I'm not certain patching osdpservices is a valid approach.

Neither can we do "removing the nova-compute-ffu from the OpenStackDataPlaneService and doing a deployment".
This is the 1st place we refer to it to be deployed, nothing to remove.
We could remove the nova-extra-config service from the existing osdpns, and make it deploying the standard nova servcice. But patching osdpns is antipattern?..